schema/me/follow_artists.js

```diff
@@ -0,0 +1,46 @@
+import gravity from '../../lib/loaders/gravity';
+import { map } from 'lodash';
+import { total as getTotal } from '../../lib/loaders/total';
+import Artist from '../artist';
+import {
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+} from 'graphql';
+
+const FollowArtists = new GraphQLObjectType({
+  name: 'FollowArtists',
+  fields: {
+    artists: {
+      type: new GraphQLList(Artist.type),
+      resolve: (data) => {
+        const artists = data.artists ? data.artists : data;
```

Comment:
This is a little weird, I'm sure theres a better way to do it. For the purposes of the module context, we resolve with an object that has a `context_type` but the normal response is just an array. We want this to resolve with artists in both cases.